Content.Client/Cargo/UI/BountyHistoryEntry.xaml.cs

```diff
@@ -0,0 +1,54 @@
+using Content.Client.Message;
+using Content.Shared.Cargo;
+using Content.Shared.Cargo.Prototypes;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+
+namespace Content.Client.Cargo.UI;
+
+[GenerateTypedNameReferences]
+public sealed partial class BountyHistoryEntry : BoxContainer
+{
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+
+    public BountyHistoryEntry(CargoBountyHistoryData bounty)
+    {
+        RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
+
+        if (!_prototype.TryIndex<CargoBountyPrototype>(bounty.Bounty, out var bountyPrototype))
+            return;
+
+        var items = new List<string>();
+        foreach (var entry in bountyPrototype.Entries)
+        {
+            items.Add(Loc.GetString("bounty-console-manifest-entry",
+                ("amount", entry.Amount),
+                ("item", Loc.GetString(entry.Name))));
+        }
+        ManifestLabel.SetMarkup(Loc.GetString("bounty-console-manifest-label", ("item", string.Join(", ", items))));
+        RewardLabel.SetMarkup(Loc.GetString("bounty-console-reward-label", ("reward", bountyPrototype.Reward)));
+        IdLabel.SetMarkup(Loc.GetString("bounty-console-id-label", ("id", bounty.Id)));
+
+        var stationTime = bounty.Timestamp.ToString("hh\\:mm\\:ss");
```

Comment:
You can use c# [verbatim strings](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim) to avoid weird string escaping issues.

```suggestion
        var stationTime = bounty.Timestamp.ToString(@"hh\:mm\:ss");
```